src/Model/UserManager.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Created by WCS.
+ * User: Kevin
+ * Date : 28/10/2020
+ */
+
+namespace App\Model;
+
+class UserManager extends AbstractManager
+{
+    const TABLE = 'user';
+
+    /**
+     *  Initializes this class.
+     */
+    public function __construct()
+    {
+        parent::__construct(self::TABLE);
+    }
+
+    public function getTable() : string
+    {
+        return self::TABLE;
+    }
```

Comment:
Constants are public, you don't need a method to access them.
Just use UserManager::TABLE

Reply:
Ok, deleted